src/web/modules/transactions/update/update.http.ts

```diff
@@ -145,6 +149,33 @@ const validateAndAddDefaults = async function validateAndAddDefaults(csv: string
         if (row.transaction_type === 'refund' && !row.parent_transaction_id) {
           return cb(null, false, 'parent_transaction_id is required when transaction_type is \'refund\'')
         }
+
+        if (['PAYMENT_STATUS_CORRECTED_TO_SUCCESS_BY_ADMIN', 'REFUND_STATUS_CORRECTED_TO_SUCCESS_BY_ADMIN'].includes(row.event_name)) {
```

Comment:
This works for `PAYMENT_STATUS_CORRECTED_TO_SUCCESS_BY_ADMIN` . For `REFUND_STATUS_CORRECTED_TO_SUCCESS_BY_ADMIN` we don't need these fields on refund itself but needed on payment record, so `REFUND_STATUS_CORRECTED_TO_SUCCESS_BY_ADMIN` can be removed

Reply:
Done